src/lib/converter/plugins/PackagePlugin.ts

```diff
@@ -98,6 +98,14 @@ export class PackagePlugin extends ConverterComponent {
                 break;
             }
 
+            if ((this.noReadmeFile || this.readmeFile) && this.packageFile) {
+                break;
+            }
+
+            if ((this.noReadmeFile || this.readmeFile) && this.packageFile) {
```

Comment:
I'm not sure how that check got duplicated. Feel free to spend 10 seconds to fix this :rofl: